IPython/core/completer.py

```diff
@@ -2073,14 +2069,22 @@ def _complete(self, *, cursor_line, cursor_pos, line_buffer=None, text=None,
         return text, _matches, origins, completions
         
     def fwd_unicode_match(self, text:str) -> Tuple[str, list]:
+        if self._names is None:
+            self._names = []
+            for c in range(0,0x10FFFF + 1):
+                try:
+                    self._names.append(unicodedata.name(chr(c)))
+                except ValueError:
+                    pass
+
         # initial code based on latex_matches() method
```

Comment:
Could you please exclude this comment? I added it while working on my last PR to keep track what I had to do, but it was intended to be erased. It's trash now.